release_notes/ocp-4-1-release-notes.adoc

```diff
@@ -0,0 +1,863 @@
+{product-title}) and using its current state to make decisions in real time.
+Advanced Operators are designed to handle upgrades seamlessly, react to failures
+automatically, and not take shortcuts, like skipping a software backup process
+to save time.
+
+[[ocp-4-1-operator-lifecycle-manager]]
+==== Operator Lifecycle Manager (OLM)
+
+This feature is now fully supported in {product-title} 4.1.
+
+The OLM aids cluster administrators in installing, upgrading, and granting
+access to Operators running on their cluster:
+
+* Includes a catalog of curated Operators, with the ability to load other Operators into the cluster
+* Handles rolling updates of all Operators to new versions
+* Supports role-based access control (RBAC) for certain teams to use certain Operators
+
+See
+xref:../applications/operators/olm-adding-operators-to-cluster.adoc#olm-adding-operators-to-a-cluster[Adding
+Operators to a cluster] for more information.
+
+[[ocp-4-1-installation-and-upgrade]]
+=== Installation and upgrade
+
+Red Hat {product-title} 4.1 has an Installer Provisioned Infrastructure (IPI),
+where the installer controls all areas of the installation process. IPI also
+provides an opinionated best practices deployment of {product-title} 4.1 for
+AWS instances only. This provides a slimmer default installation, with
+incremental feature buy-in through OperatorHub.
+
+Upgrading from 3.x to 4.1 is currently not available. You must perform a new
+installation of {product-title} 4.1.
+
+Easy, over-the-air upgrades for asynchronous z-stream releases of
+{product-title} 4.x is available. Cluster administrators can upgrade using the
+*Cluster Settings* tab in the web console. See
+xref:../updating/updating-cluster.adoc#updating-cluster[Updating a cluster] for
+more information.
+
+[[ocp-4-1-operator-hub]]
+==== OperatorHub
+
+OperatorHub is available to administrators and helps with easy discovery and
+installation of all optional components and applications. It includes offerings
+from Red Hat products, Red Hat partners, and the community.
+
+.Features provided with base installation and OperatorHub
+[cols="3",options="header"]
+|====
+|Feature |New installer |OperatorHub
+
+|Console and authentication
+|* [x]
+| -
+
+|Prometheus cluster monitoring
+|* [x]
+| -
+
+|Over-the-air updates
+|* [x]
+| -
+
+|Machine management
+|* [x]
+| -
+
+|Optional service brokers
+| -
+|* [x]
+
+|Optional {product-title} components
+| -
+|* [x]
+
+|Red Hat product Operators
+| -
+|* [x]
+
+|Red Hat partner Operators
+| -
+|* [x]
+
+|Community Operators
+| -
+|* [x]
+
+|====
+
+[[ocp-4-1-storage]]
+=== Storage
+
+Storage support in {product-title} 4.1 is the same as {product-title} 3.11 with
+the exception of the following available in Technology Preview: EFS (CSI Driver
+handled via Amazon), Manilla provisioner/operator, and Snapshot.
+
+[[ocp-4-1-CSI]]
+==== Container Storage Interface
+
+In {product-title} 4.1, Container Storage Interface (CSI) is now generally
+available.
```

Comment:
@jsafrane  Is it true ?

Reply:
@gnufied Could you take a look ?

Reply:
CSI is not yet generally available with Openshift 4.1, because we aren't yet shipping standard sidecar containers. CSI is enabled and more like - DYI config but isn't GAed yet. cc @childsb @jsafrane

Reply:
Yes, CSI is **NOT** GA in 4.1

Reply:
Thank you for confirming. I am removing this.